fall18/p3/project.py

```diff
@@ -0,0 +1,96 @@
+import csv
+
+class Project():
+    hdata={}
+    total=0
+    def __init__(self):
+        with open('hurricanes.csv') as csvDataFile:
+            csvReader = csv.reader(csvDataFile)
+            iterator=0
+            for row in csvReader:
+                if iterator ==0:
+                    iterator+=1
+                    continue
+                self.hdata[iterator]=row
+                iterator+=1
+            self.total=iterator-1
+
+    """ Function that takes in record number and returns the name of the hurricane"""

```

Comment:
The doc string has to go after the def line in Python, otherwise tricks like methodname.__doc__ don't tell you anything.